api/authorization.go

```diff
@@ -88,6 +89,11 @@ func authorization(authUrl string) middleware {
 			r = r.WithContext(ctx)
 		}
 
+		if isCallerAdmin := authRes.Header.Get("X-Livepeer-Is-Caller-Admin"); isCallerAdmin != "" {
```

Comment:
We should parse `isCallerAdmin` as a boolean with `strconv.ParseBool` (or sth like that)